Makefile

```diff
@@ -0,0 +1,36 @@
+IXO_ROOT		   ?= https://github.com/ixofoundation/ixo-blockchain
+IXO_RELEASE	   ?= v1.6.0
+IXO_DST_FOLDER ?= ixo-blockchain-${IXO_RELEASE}
+GOOS 					 ?= linux
+GOARCH				 ?= amd64
+
+all: clone build-ixod
+
+clone:
+	@echo "Clonning repo ${IXO_ROOT}"
+	test -d ${IXO_DST_FOLDER} || \
+	git clone ${IXO_ROOT} ${IXO_DST_FOLDER}
+	cd ${IXO_DST_FOLDER} && \
+	git fetch --all && \
+	git checkout ${IXO_RELEASE}
+
+build-ixod: clone
+	docker run \
+	-v "$(CURDIR)/${IXO_DST_FOLDER}:/app" \
+	-v "$(CURDIR)/bin:/go/bin/" \
+	-e GOARCH=${GOARCH} \
+	-e GOOS=${GOOS} \
+	--workdir "/app" golang:1.16 make install
```

Comment:
if possible, it would be nice to have golang version as a variable too, and read from both Dockerfile and here (not sure it possible but sure is better)

Reply:
@sibuser and that is how you can do it: https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact